src/Analysis/Engine/Test/AstAnalysisTests.cs

```diff
@@ -810,7 +811,17 @@ class BankAccount(object):
                 DatabasePath = TestData.GetAstAnalysisCachePath(configuration.Version, true),
                 UseExistingCache = false
             });
-            var modules = ModulePath.GetModulesInLib(configuration.PrefixPath).ToList();
+
+            IEnumerable<ModulePath> modules;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
+                modules = ModulePath.GetModulesInLib(configuration.PrefixPath).ToList();
+            } else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) {
+                var libPath = $"/Library/Frameworks/Python.framework/Versions/{configuration.Version}/lib/python{configuration.Version}";
```

Comment:
Yeah, I know. However, tests use fixed configuration rather than discovery. In general, LS does not discover Python, the path is provided by the client app (see question in https://github.com/Microsoft/python-language-server/issues/133). So I can either add additional paths and probe them or we simply require certain setup for tests (big VS does the latter).

Reply:
Let me update `CONTRIBUTING`

Reply:
We have this done in `PythonInstallPathResolver`.

Reply:
I will need to clean up interpreter configuration code quite a bit, stay tuned

Reply:
Yeah, I'm not entirely convinced that PythonInstallPathResolver does the right thing in all cases. For example, the unix one goes out to get `sys.prefix` to set `PrefixPath`, which on my Arch machine turns out to be `/usr`, which isn't going to be helpful whatsoever when searching for Python libraries. On macOS, `sys.prefix` is more useful. Using `inspect` to ask Python where some stdlib package lives seems to be more reliable, at least for resolving that specific path.

I feel like the whole interpreter configuration thing (and ModulePath) needs to be reframed into a better view on the interpreter in terms of search paths and so on. Mikhail and I talked about this just after lunch.

Reply:
There is some Windows and UI related code I want to get rid of